File: ts-sdk/src/sdk/models/shared/comment.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// Comment
/** 
 * A Comment on an Article or Podcast Episode
**/
export class Comment extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=created_at" })
  createdAt?: Date;

  @SpeakeasyMetadata({ data: "json, name=id_code" })
  idCode?: string;

  @SpeakeasyMetadata({ data: "json, name=image_url" })
  imageUrl?: string;

  @SpeakeasyMetadata({ data: "json, name=type_of" })
  typeOf?: string;
}

File: go-sdk/pkg/models/shared/comment.go
package shared

import (
	"time"
)

// Comment
// A Comment on an Article or Podcast Episode
type Comment struct {
	CreatedAt *time.Time `json:"created_at,omitempty"`
	IDCode    *string    `json:"id_code,omitempty"`
	ImageURL  *string    `json:"image_url,omitempty"`
	TypeOf    *string    `json:"type_of,omitempty"`
}
